### ISSUE TITLE
Error: Cannot find module 'react-google-places-suggest'

### ISSUE THREAD

**Issue by LucaAndrei** [xuopled/react-google-places-suggest#7]
I have installed this package in my project but when I try to import it into one of my React classes, it says that it cannot find the module.
I have searched for the folder in the node_modules folder and it is there.
Is there anything else I should do?

**Comment by xuopled** [comment]
It is very strange. Could you give me a bit of code ?

**Comment by todda00** [comment]
I got this same issue, figured out it is happening because of an erroneous property in the package.json file: 

https://github.com/xuopled/react-google-places-suggest/blob/master/package.json#L23
`"browser": "./dist/index.js",`

This is replacing the main entrypoint with /dist/index.js , which does not exist. 

Remove this entry, or correct it to lib/index.js to resolve the issue. I manually modified the package's file and removed the browser property in my project and it works now, but work with other developers so patiently awaiting the fix on this project so I don't have to tell them to manually remove the line. 

Info on browser property: https://github.com/defunctzombie/package-browser-field-spec#spec
